MetaMorpheus/GuiFunctions/MetaDraw/MetaDrawLogic.cs

```diff
@@ -795,6 +796,7 @@ public void CleanUpSpectraFiles()
                 {
                     connection.Value.CloseDynamicConnection();
                 }
+                Thread.Sleep(1000);
```

Comment:
let's put a comment here explaining what happens if you don't have this sleep.

Reply:
Seconded

Reply:
Comment added